signing_test.go

```diff
@@ -521,3 +521,39 @@ func TestSignerExtraHeaderInclusion(t *testing.T) {
 		t.Errorf("Mismatch in extra headers: %#v", object.Signatures[0].Header.ExtraHeaders)
 	}
 }
+
+func TestSignerB64(t *testing.T) {
+	signingKey, verificationKey := GenerateSigningTestKey(RS256)
+
+	opts := &SignerOptions{
+		ExtraHeaders: map[HeaderKey]interface{}{
+			"b64": false,
+		},
+	}
```

Comment:
This seems somewhat non-obvious, would it be better to have this be a first-class option in SignerOptions and have the signing code add the header if that's selected?